examples/nemo/nemo.yaml

```diff
@@ -0,0 +1,62 @@
+# Distributed training a BERT model with Nvidia NeMo
+#
+# Finetunes a BERT-like model on the GLUE CoLA task. Uses the NeMo toolkit
+# to train across multiple nodes, each node having a V100 GPU.
+#
+# Uses glue_benchmark.py script from the NeMo examples:
+# https://github.com/NVIDIA/NeMo/blob/2ce45369f7ab6cd20c376d1ed393160f5e54be0c/examples/nlp/glue_benchmark/glue_benchmark.py
+#
+# Usage:
+#   sky launch -c nemo nemo.yaml
+#   sky down nemo
+
+resources:
+  accelerators: V100:1
+
+num_nodes: 2
+
+setup: |
+  conda activate nemo
+  if [ $? -eq 0 ]; then
+      echo "conda env exists"
+  else
+      conda create -y --name nemo python==3.10.12
+      conda activate nemo
+  
+      # Install PyTorch
+      pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
+      
+      # Install nemo
+      sudo apt-get update
+      sudo apt-get install -y libsndfile1 ffmpeg
+      pip install Cython
+      pip install nemo_toolkit['all']
+    
+      # Clone the NeMo repo to get the examples
+      git clone https://github.com/NVIDIA/NeMo.git
+      
+      # Download GLUE dataset
+      wget https://gist.githubusercontent.com/W4ngatang/60c2bdb54d156a41194446737ce03e2e/raw/70e86a10fbf4ab4ec3f04c9ba82ba58f87c530bf/download_glue_data.py
+      python download_glue_data.py --data_dir glue_data --tasks CoLA
+  fi
+
+run: |
+  conda activate nemo
+  
+  # Get the number of nodes and master address from SkyPilot envvars
+  num_nodes=`echo "$SKYPILOT_NODE_IPS" | wc -l`
+  master_addr=`echo "$SKYPILOT_NODE_IPS" | head -n1`
+  
+  # Run glue_benchmark.py
+  python -m torch.distributed.run \
+    --nproc_per_node=1 \
```

Comment:
We can use `$SKYPILOT_NUM_GPUS_PER_NODE` here. : )

Reply:
Great catch! fixed now.